antlr/SimplifiedCql.g4

```diff
@@ -183,7 +183,12 @@ condition
 
 // Note: custom index expressions not supported
 whereClause
-    : K_WHERE relation ( K_AND relation )*
+    : K_WHERE relation ( logicalOperator relation )*
+    ;
+
+logicalOperator
+    : K_AND
+    | K_OR
```

Comment:
these are basically the only changes to the production code right? just to make sure I'm not missing anything. The rest of the diff is just test code?

Reply:
And the one below with `:`. Production code changes were only for syntax definition.